### ISSUE TITLE
☂️ Translate the website into Portuguese

### ISSUE THREAD

**Issue by CookieDasora** [biomejs/biome#1065]
### Description

If you would like to contribute to a specific page, please leave a comment indicating the page you would like to translate and create a PR.
If there is any error, please notify here in the comments.

- [x] Translate sidebar @CookieDasora 
- [x] Translate `README.md` @ImBIOS 
- [x] Translate the files inside `src/content/docs`
  - [x] `index.md` @CookieDasora 
  - [x] `404.md` @CookieDasora 
  - [x] `analyzer/index.mdx` @Levieber 
  - [x] `formatter/index.mdx` @danspinola 
  - [x] `formatter/option-philosophy.md` @danspinola 
  - [x] `formatter/differences-with-prettier.md` @ImBIOS 
  - [x] `linter/index.mdx` @CookieDasora 
  - [x] `guides/big-projects.mdx` @CookieDasora 
  - [x] `guides/getting-started.mdx` @CookieDasora 
  - [x] `guides/how-biome-works.mdx` @CookieDasora 
  - [x] `guides/integrate-in-editor.mdx` @CookieDasora 
  - [x] `guides/manual-installation.mdx` @CookieDasora 
  - [x] `internals/architecture.mdx` @ImBIOS 
  - [x] `internals/credits.mdx` @CookieDasora 
  - [x] `internals/language-support.mdx` @CookieDasora 
  - [x] `internals/philosophy.mdx` @Levieber 
  - [x] `internals/versioning.mdx` @Levieber 
  - [x] `recipes/continuous-integration.mdx` @CookieDasora 
  - [x] `reference/vscode.mdx` @ImBIOS 

The following files **don't** need to be translated, since they are auto-generated
- `internals/changelog.mdx`
- `linter/rules/*.mdx`
- `reference/configuration.mdx` (will be auto-generated in #825 )
- `reference/cli.mdx`
- `blog/*.mdx` (low priority)

**Comment by Levieber** [comment]
I want to take `analyzer/index.mdx`

**Comment by danspinola** [comment]
A few questions:

\- Which props of the frontmatter should be translated?  
\- Which gender declination to use for "Biome"? (masculine or feminine)  
\- Are we supposed to translate "toolchain"? A quick search on [Linguee](https://www.linguee.pt/portugues-ingles/search?source=auto&query=toolchain) shows three options:

*   Not translate. Keep "toolchain" as is in the translated text.
*   Translate as "(a) cadeia de ferramentas"
*   Translate as "(o) conjunto de ferramentas"

**Comment by CookieDasora** [comment]
> Which props of the frontmatter should be translated?

I didn't quite understand your question, but I'll answer based on what I understood. We should translate only the `.mdx` files located within `/src/content/docs`; this **does not** include automatically generated files, such as linter rules, for example. Other interface components, like the sidebar, will not be translated.

> Which gender declination to use for "Biome"? (masculine or feminine)

I believe the masculine gender is the most appropriate for Biome, such as "do Biome" and "o Biome".

> Are we supposed to translate "toolchain"?

@victor-teles suggested translating 'toolchain,' and I applied that suggestion in commit 7ee1d34. I believe it would be good to translate it as well, as it has also been translated in other languages.

**Comment by danspinola** [comment]
Front matter is the yaml-looking part at the top of the markdown. But I can see in the commit you mentioned that the front matter was translated.

**Comment by CookieDasora** [comment]
Oh, yes! Didn't know about that, sorry!

**Comment by danspinola** [comment]
I can take both `formatter/index.mdx` and `formatter/option-philosophy.md`.

**Comment by Levieber** [comment]
I want to take `internals/philosophy.mdx`

**Comment by Levieber** [comment]
I want to take `internals/versioning.mdx`

**Comment by ImBIOS** [comment]
Let me take `reference/vscode.mdx`

**Comment by victor-teles** [comment]
@ImBIOS all yours :)

**Comment by ImBIOS** [comment]
While waiting for feedback in #1181 Can I work on `internals/architecture.mdx` ? Do you prefer I use my existing PR, or creating a new PR is better? In my opinion, it'd be better if I made a new PR, each PR track a different translation review. So, each file can be reviewed in parallel.

**Comment by victor-teles** [comment]
@ImBIOS 

> While waiting for feedback in https://github.com/biomejs/biome/pull/1181 Can I work on internals/architecture.mdx ? Do you prefer I use my existing PR, or creating a new PR is better? In my opinion, it'd be better if I made a new PR, each PR track a different translation review. So, each file can be reviewed in parallel.

Sure i'll assign to you.
A new PR would be good.

>  Wait... I just realized, does "all yours" mean I assigned all remaining tasks? If that's true, what a happy day in my life as a language enthusiast.

Sorry for the misunderstanding, I was just referring to 'reference/vscode.mdx'!
But we just have 3 pages remaining, you can take it if you want

**Comment by ImBIOS** [comment]
Let me take `formatter/differences-with-prettier.md` and `README.md`, I planned to make a single PR for each doc, for easier review.

**Comment by ImBIOS** [comment]
Other than already translated docs, I can read and write in Bahasa Indonesia (native), International Arabic, Turkish, International Russian, and Javanese/Basa Jawa (native) -don't worry, I'm not a _hikikomori_ or _NEET_ or _shut-in_ person-. Can we open another translation mission? Let's make the docs blow over any search engine in any language or region.

**Comment by victor-teles** [comment]
@danspinola Are you still interested in working on formatter/index.mdx and formatter/option-philosophy.md pages?

**Comment by danspinola** [comment]
mate, I opened a PR days ago.

**Comment by victor-teles** [comment]
Have you opened it to Biome's main repo? I can't see it in the pull request list

https://github.com/biomejs/biome/pulls

**Comment by danspinola** [comment]
No, I opened to CookieDasora's fork. https://github.com/CookieDasora/biome-portuguese-translation/pull/1

**Comment by ematipico** [comment]
@danspinola you should open the PR to this repository :)

**Comment by ematipico** [comment]
Hi @danspinola , can you open a PR to this repository so we can review and merge your contribution?

**Comment by danspinola** [comment]
Done.

**Comment by victor-teles** [comment]
All planned pages are now in PT :)
Thank you to everybody who helped with the Portuguese translation! 🇵🇹 🎉
